src/components/Info.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import info from './icons/info.svg';
+import './Info.css';
+
+export default class Info extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      show: false
+    };
+    this.toggleInfo = this.toggleInfo.bind(this);
+  }
+
+  toggleInfo() {
+    this.setState({
+      show: !this.state.show
+    });
+  }
+
+  render() {
+    const displayClass = this.state.show ? 'Show-info' : 'Hide-info';
```

Comment:
The classes you had before were fine -- ie, `.show` and `.hide`. What is needed is for the element that they're being applied to have its own class separate to that (`Info-text`, for example). Then, if show/hide is toggled, it would be a state of `.Info-text .show` which is easier to read/reason about.

Reply:
Done.